src/game/default/g_cmd.c

```diff
@@ -942,12 +1012,13 @@ static void G_Ready_f(g_entity_t *ent) {
 	}
 
 	if (ent->client->locals.persistent.ready) {
-		gi.ClientPrint(ent, PRINT_HIGH, "You're already ready\n");
+		G_Unready_f(ent);
```

Comment:
Is typing `"ready"` multiple times how Q2 duel mods handle this, or should we map `"unready"` explicitly as well? The way this is now, if I forget that I'm ready, and type "ready", I've just un-readied myself. This is more of a question than a suggestion.